src/components/wallet/manage/KeyRow.vue

```diff
@@ -95,6 +95,20 @@
             <div class="header">
                 <div></div>
                 <div>
+                    <div class="addressBalance bal_cols" v-if="updatedBalance === '0'">
+                        <p>{{ $t('keys.empty') }}</p>
+                    </div>
+                    <div class="addressBalance bal_cols" v-else>
+                        <p>This key has:</p>
+                        <div class="bal_rows">
+                            <p>
+                                {{ updatedBalance }}
+                                <b>FLR</b>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <!-- <div v-else>
                     <p v-if="Object.keys(balances).length === 0" class="balance_empty">
```

Comment:
Instead of commenting this code, update the variable here through which we are getting balance